sql/core/src/main/java/org/apache/spark/sql/vectorized/ArrowColumnVector.java

```diff
@@ -55,164 +43,82 @@ public void close() {
     if (childColumns != null) {
       for (int i = 0; i < childColumns.length; i++) {
         childColumns[i].close();
+        childColumns[i] = null;
```

Comment:
Is it OK not to call `close()` while `ColumnVector.close()` is provided?

Reply:
what do you mean? `ColumnVector.close` is a required interface.